pkg/scripts/download.go

```diff
@@ -44,14 +44,14 @@ func NewDownloader(proxyConfig *api.ProxyConfig) Downloader {
 // Returns information a boolean flag informing whether any download/copy operation
 // happened and an error if there was a problem during that operation
 func (d *downloader) Download(url *url.URL, targetFile string) (*api.SourceInfo, error) {
-	schemeReader := d.schemeReaders[url.Scheme]
+	r := d.schemeReaders[url.Scheme]
```

Comment:
Shadowing. `schemeReader` is a top-level declared type.

Within this short context, calling a thing of type `schemeReader`, coming out of `d.schemeReaders[url.Scheme]`, doesn't seen to me "unreadable" when looking at it with a "Go-philosophy hat".

https://github.com/golang/go/wiki/CodeReviewComments#variable-names